### ISSUE TITLE
some question

### ISSUE THREAD

**Issue by jackiezhang11** [JierunChen/FasterNet#35]
Can I use 416*416 picture as input？ what about 448*448  in detection without pre-training model?Could fasternet as YOLOV3 backbone?

**Comment by JierunChen** [comment]
@jackiezhang11 Hi, both 416 and 448 can be the input size. The detection performance would degrade without pre-training. And yes, FasterNet should be applicable as a backbone for YOLOV3.

**Comment by jackiezhang11** [comment]
Thank you for quickly apply.Could you give me an idea to make FasterNet as YOLOV3's backbone.Is it altering network in Pytorch and altering cfg file in darknet?I don't konw is it right.If you konw,Could you tell me details to alter?

**Comment by jackiezhang11** [comment]
Use Fasternet as backbone would improve accuracy.Could you tell me the reason ?such as compared with  YOLOV3.

**Comment by zhaigz** [comment]
> Use Fasternet as backbone would improve accuracy.Could you tell me the reason ?such as compared with YOLOV3.

Hi, I have been following how to replace FasterNet with ResNet recently. Have you implemented it yet? How is the effect? Can we discuss it? Thank you very much.
